api/v1/runtime/runtime_agent_api.proto

```diff
@@ -63,6 +63,8 @@ message Event {
   EventType event_type = 1;
   uint64 timestamp = 2; // Stored as unix timestamp in nanoseconds.
   string process_name = 3;
+  uint32 process_pid = 15;
```

Comment:
I wonder if it makes sense to move the fields we need to identify a container into a `ProcessIdentity` struct to make it explicit.

Reply:
Something like this?

```proto
message ProcessIdentity {
  string container_id = 1;
  string process_pid = 2;
  string process_start_time = 3;
}
```

The only problem I see is the `container_id` is already part of the `Event` proto message so we need to set it in both places to make it backward compatible:

https://github.com/castai/kvisor/blob/38e892a5d95cd413b34f4b72828fda389444c088/api/v1/runtime/runtime_agent_api.proto#L72

Reply:
The `container_id` is not really part of the process identity though. I know we need it to globally identify a process, but to identify a process on a host, `process_pid` and `process_start_time` are enough.

Reply:
We also have `workload_name`, `workload_kind` and `workload_uid` in the `Event` message instead of having a `WorkloadIdentity` field so I'm not sure if adding `ProcessIdentity` is aligned with the rest of the API spec

Reply:
`workload_uid` is enough to uniquely identify a workload though, whereas a `PID` is not due to reuse, which can happen quite often.

Reply:
Added in https://github.com/castai/kvisor/pull/381/commits/caf4aaba59604a4437ab067f650a27d6cd82ee15